.github/workflows/upgrade-branch-to-version.yaml

```diff
@@ -1,51 +0,0 @@
-name: upgrade-branch-to-version
```

Comment:
Why completely delete this? Did it not actually work?

Reply:
The shell script was removed as it was the old version, but I can easily add this with a pnpm command

Reply:
I see no past runs, so was def not popular. Guess we can rebuild when/if we need it.